### DISCUSSION TITLE
Add Simple Wizard to Relational Field Setup

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3277] [Feature Requests]
Similar to Translations, we should have an easy wizard for configuring these relational fields for those who don't care about specific junction/table names. This is also where we would add an option for adding a "Sort" column (ref: https://github.com/directus/directus/issues/3275)

**Comment by rijkvanzanten** [comment]
![CleanShot 2020-12-02 at 18 00 54](https://user-images.githubusercontent.com/9141017/100941902-6e167080-34c8-11eb-94e5-09d81f115923.png)

![](https://media.giphy.com/media/Ll37bXYmQecEdBpmmM/giphy.gif)

**Comment by rijkvanzanten** [comment]
This has since been added!
